frontend/src/container/PublicProfile.jsx

```diff
@@ -374,8 +374,9 @@ export class PublicProfile extends React.Component {
   isPublicProfileEditable = (connectedUser, publicProfileId, profileObject) => {
     const isConnectedUserOnHisOwnProfile = connectedUser.userId === publicProfileId
     const isUserAdmin = connectedUser.profile === profileObject.administrator.slug
+    const isInformationSchemaObjectEmpty = Object.keys(this.state.informationSchemaObject).length === 0
```

Comment:
shouldn't you also test the personnalDataSchemaObject ?

Also, I'd like this function to stay pure

And finally, you could use and early return if isInformationSchemaObjectEmpty is true
It would simplify the final return

Reply:
When you said `personalDataSchemaObject` I think you meant `informationDataSchema`, correct if I'm wrong. At this case, the object can be empty even if `informationSchemaObject` is not empty. The first time that the user go to the profile, for example.

I'll change the rest.

Reply:
no, I mean that `isPublicProfileEditable` is used by both form (information and personal page) and I suppose it should return a result for each form based on its own schema

Reply:
Information form: https://github.com/tracim/tracim/blob/develop/frontend/src/container/PublicProfile.jsx#L448

Personal page form: https://github.com/tracim/tracim/blob/develop/frontend/src/container/PublicProfile.jsx#L472

Reply:
ah, ok, will fix